src/main/java/com/volmit/iris/engine/object/IrisTreeType.java

```diff
@@ -0,0 +1,69 @@
+package com.volmit.iris.engine.object;
```

Comment:
Instead, of making your own tree type, Literally just use TreeType from bukkit. This would allow support for new tree types without having to change any code. There's no need to make a delegate array with a wrapper.